src/test/java/uk/gov/dhsc/htbhf/local/LocalWebDriverWrapper.java

```diff
@@ -68,6 +68,8 @@ private WebDriver buildWebDriver() {
             case "chrome":
                 ChromeOptions chromeOptions = new ChromeOptions();
                 chromeOptions.setHeadless(headless);
+                // try to avoid "(unknown error: DevToolsActivePort file doesn't exist)": https://stackoverflow.com/a/56758428
+                chromeOptions.addArguments("--no-sandbox", "--disable-dev-shm-usage");
```

Comment:
Is there something similar for the Firefox driver? Do we care?

Reply:
I think the issue is specific to chrome